components/RecurringEvents.vue

```diff
@@ -45,42 +42,42 @@
 import Zondicon from 'vue-zondicons'
 
 export default {
-  props: ['title', 'url'],
+  props: ['title', 'url', 'announcement'],
   components: { Zondicon },
   data() {
     return {
       events: [
```

Comment:
Maybe we can just plug `$t('recurring_events')` in here, and manage the recurring events in the translation files.